client/components/progress-indicator/style.scss

```diff
@@ -1,223 +0,0 @@
-// ==========================================================================
-// .progress-indicator
-//
-// Includs a spinner and the ability to have specific progress-indicator__percentages using the
-// same elements.
-//
-// Check out this demo to understand how this works:
-// http://codepen.io/MichaelArestad/pen/2740d550eaa89cef6c753316595298b7
```

Comment:
> Would be great to know what this component looks like though before removal.

It seems like this demo partly answers that question.

Reply:
Ah! I missed that. I suppose the Spinner has replaced this component? https://wpcalypso.wordpress.com/devdocs/design/spinner

I'll cc @MichaelArestad

Reply:
It seems I was mistaken, the Spinner is just a loading animation, and the progress indicator shows status messages.

I also missed that site indicator was using it, so we shouldn't remove the component.